W-18439719 b2c connectors note (#2883)
mulesoft/docs-connectors: connectors-home/modules/ROOT/partials/b2c-connectors-deprecated.adoc

// Used in Commerce Cloud B2C Shop and Commerce Cloud B2C Data Connector Documentation for a note saying to use HTTP Connector instead

> NOTE: There will be no further updates or new versions to this connector. Use xref:http-connector::index.adoc[Anypoint Connector for HTTP (HTTP Connector)] instead. Refer to the https://developer.salesforce.com/docs/commerce/commerce-api/guide/migrate-mulesoft-http-connector.html[migration guide]. 

Reviewer: ```suggestion
NOTE: No further updates or new versions are planned for this connector. Use xref:http-connector::index.adoc[Anypoint Connector for HTTP (HTTP Connector)] instead. Refer to the https://developer.salesforce.com/docs/commerce/commerce-api/guide/migrate-mulesoft-http-connector.html[migration guide]. 
```
